oauth2-proxy.yaml

```diff
@@ -46,7 +46,51 @@ update:
     strip-prefix: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - dex
+        - curl
   pipeline:
-    # AUTOGENERATED
     - runs: |
-        oauth2-proxy --version
+        cat <<EOF > dex.yaml
+        issuer: http://127.0.0.1:5556/dex
+
+        connectors:
+        - type: mockCallback
+          id: mock
+          name: Example
+
+        storage:
+          type: sqlite3
+          config:
+            file: dex.db
+
+        web:
+          http: 0.0.0.0:5556
+
+        staticClients:
+        - id: oauth2-proxy
+          redirectURIs:
+          - 'http://127.0.0.1:4180/oauth2/callback'
+          name: 'oauth2-proxy'
+          secret: proxy
+        EOF
+
+        dex serve dex.yaml &
+        sleep 5
```

Comment:
This seems like a good place to use `test/daemon-check-output` so that package tests don't fail if we time things wrong